src/stan/io/stan_csv_reader.hpp

```diff
@@ -256,14 +256,14 @@ namespace stan {
             break;
 
           if (comment_line) {
-            if (line.find("(Warm-up)") != std::string::npos) {
-              int left = 17;
-              int right = line.find(" seconds");
+            if (line.find("Warmup") != std::string::npos) {
```

Comment:
This is just an update to the csv_reader because the
writers change the CmdStan output slightly.  Not sure
what the general utility would be.

Really this needs to get pulled into CmdStan in a following
update.

On Jul 18, 2015, at 5:57 AM, Daniel Lee <notifications@github.com> wrote:

> In src/stan/io/stan_csv_reader.hpp:
> 
> > @@ -256,14 +256,14 @@ namespace stan {
> >              break;
> >  
> >            if (comment_line) {
> > -            if (line.find("(Warm-up)") != std::string::npos) {
> > -              int left = 17;
> > -              int right = line.find(" seconds");
> > +            if (line.find("Warmup") != std::string::npos) {
> @betanalpha, can we split this into its own pull request so it can patch develop now? This seems useful.
> 
> —
> Reply to this email directly or view it on GitHub.
>